docs/pages/components/swiper.mdx

```diff
@@ -19,238 +19,120 @@ import {
-          />
-        </Swiper.Slide>
-      </Swiper>
-    </>
-  )
-}
-```
-
-## Autoplay
+## With pagination
 
-Providing `autoplay` lets the slides advance automatically. Pass `duration` to control the amount of time each slide shows for.
+By providing the optional `withPagination` with `[true, true]`, you can deactivate the pagination on mobile/tablet and on desktop.
 
 ```jsx
 function() {
-  const swiper = useSwiper({
-    autoplay: true,
-    loop: true,
-    nextButton: <RightIcon size="lg" />,
-    prevButton: <LeftIcon size="lg" />
-  })
-
   return (
-    <Swiper {...swiper} h={400}>
-      <Swiper.Slide textAlign="center">
-        <img
-          style={{ maxWidth: '100%', maxHeight: '100%' }}
-          src="https://images.unsplash.com/photo-1564460549828-f0219a31bf90?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80"
-        />
-      </Swiper.Slide>
-      <Swiper.Slide textAlign="center">
-        <img
-          style={{ maxWidth: '100%', maxHeight: '100%' }}
-          src="https://images.unsplash.com/photo-1575489272413-cb506258027e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80"
-        />
-      </Swiper.Slide>
-      <Swiper.Slide textAlign="center">
-        <img
-          style={{ maxWidth: '100%', maxHeight: '100%' }}
-          src="https://images.unsplash.com/photo-1541959833400-049d37f98ccd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80"
-        />
-      </Swiper.Slide>
+    <Swiper h={400} withPagination={[true, true]}>
+      <img
+        style={{ maxWidth: '100%', maxHeight: '100%', objectFit: 'cover' }}
+        src="https://images.unsplash.com/photo-1564460549828-f0219a31bf90?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80"
+      />
+      <img
+        style={{ maxWidth: '100%', maxHeight: '100%', objectFit: 'cover' }}
+        src="https://images.unsplash.com/photo-1575489272413-cb506258027e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80"
+      />
+      <img
+        style={{ maxWidth: '100%', maxHeight: '100%', objectFit: 'cover' }}
+        src="https://images.unsplash.com/photo-1541959833400-049d37f98ccd?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80"
+      />
     </Swiper>
   )
 }
 ```
 
-## With custom pagination items
+## Without arrow buttons
 
-By providing the optional `renderPaginationItem` you can customise the defaults pagination bullets and render your own component. This callback takes a `props` param which is an object containning:
-
-- `idx`: the current pagination item
-- `pageIdx`: the currently selected slide (which you can use to show the active state for your pagination items)
-- `onClick`: callback to pass to your custom bullet
+By providing the optional `withArrows` with `[false, false]`, you can hide the navigation arrows on mobile/tablet and desktop.
```

Comment:
I'm wondering for all these props if it wouldn't be clearer to have an object?
Like this:
```
withArrows = { mobile: false, desktop: false }
```
In my opinion this is more explicit but I don't know if it's more complex

Reply:
Yep I agree, I changed all the props with an array for an object, thanks!

Reply:
🔥  thank you!